tensorflow/lite/experimental/micro/micro_interpreter.cc

```diff
@@ -299,4 +299,42 @@ TfLiteTensor* MicroInterpreter::tensor(size_t index) {
   return &context_.tensors[index];
 }
 
+std::pair<TfLiteNode, const TfLiteRegistration*> MicroInterpreter::node_and_registration(int node_index) {
```

Comment:
Wondering if there is a way to avoid std::pair dependency in the interpreter code? Probably either define a struct for std::pair or subclass/friend class?